chaoscenter/authentication/pkg/entities/project.go

```diff
@@ -10,9 +10,13 @@ type Project struct {
 }
 
 type Owner struct {
-	UserID   string `bson:"user_id" json:"userID"`
-	Username string `bson:"username" json:"username"`
+	UserID        string     `bson:"user_id" json:"userID"`
```

Comment:
make it memory efficient. Ref: https://towardsdev.com/golang-writing-memory-efficient-and-cpu-optimized-go-structs-62fcef4dbfd0

Reply:
Username is an mandatory field, making it pointer does not makes sense

Reply:
hey @SarthakJain26 better align give many more suggestions to save memory should I modify them too or stick with the schema I wrote.